src/main/java/org/traccar/protocol/Gt06ProtocolDecoder.java

```diff
@@ -978,7 +993,8 @@ private Object decodeBasic(Channel channel, SocketAddress remoteAddress, ByteBuf
                 buf.readUnsignedByte(); // validity
             }
 
-            if (buf.readableBytes() == 3 + 6 || buf.readableBytes() == 3 + 4 + 6) {
+            if ((buf.readableBytes() == 3 + 6 || buf.readableBytes() == 3 + 4 + 6)
+                    && !("NT20".equals(model) && type == MSG_GPS_LBS_2)) {
```

Comment:
Couldn't we just move the block that's now below to above. And it will solve this problem?